public-ui/src/app/data-browser/views/survey-chart/survey-chart.component.ts

```diff
@@ -100,6 +100,11 @@ export class SurveyChartComponent implements OnInit {
           break;
       }
     }
+    if (q.selectedAnalysis === undefined) {
+      this.displayGraphErrorMessage = true;
+    } else {
+      this.displayGraphErrorMessage = false;
+    }
```

Comment:
Nit: I think this block could be simplified to a single line:
`this.displayGraphErrorMessage = q.selectedAnalysis === undefined;`